doc/api/api_changes/2017-07-14-IT_qhull_large_offset.rst

```diff
@@ -0,0 +1,6 @@
+Improved Delaunay triangulations with large offsets
```

Comment:
@dstansby @QuLogic My apologies, I thought I'd responded to this ages ago but evidently I didn't. I have amended the API change text now.